test/util/bspNet/test-api.ts

```diff
@@ -258,7 +258,8 @@ export class BspNetTestApi implements AsyncDisposable {
        * @param expectedExts - Optional param to specify the number of expected extrinsics.
        * @returns A promise that resolves when a BSP has confirmed storing a file.
        */
-      bspStored: (expectedExts?: number) => Waits.waitForBspStored(this._api, expectedExts),
+      bspStored: (expectedExts?: number, bspAccount?: string) =>
```

Comment:
I'd suggest using the `Address` type from `@polkadot/types/interfaces` here, instead of a plain `string`.